.github/workflows/gke.yaml

```diff
@@ -99,8 +99,7 @@ jobs:
           kubectl logs --timestamps -n kube-system job/cilium-cli
           kubectl exec -n kube-system job/cilium-cli -- cilium status
           kubectl get pods --all-namespaces -o wide
-          curl -sLO https://github.com/cilium/cilium-sysdump/releases/latest/download/cilium-sysdump.zip
-          python cilium-sysdump.zip --output cilium-sysdump-out
+          cilium sysdump --output-filename cilium-sysdump-out
```

Comment:
It appears that in this and maybe other workflows `cilium` (cilium-cli) is not installed locally as it is only used in the cloud via job/cilium-cli. This likely means that sysdump also needs to be run as a k8s job?

Now this fails like:
```
/home/runner/work/_temp/ff9b05e1-e1d4-4ae0-b1fb-1f9c7fc6626b.sh: line 4: cilium: command not found
```
and
```
Warning: No files were found with the provided path: cilium-sysdump-out.zip. No artifacts will be uploaded.
```

Reply:
@jrajahalme i think this pr from @nbusseneau will address the issue https://github.com/cilium/cilium-cli/pull/423